src/NServiceBus.Core/Licensing/LicenseExpiredForm.cs

```diff
@@ -80,19 +84,62 @@ void browseButton_Click(object sender, EventArgs e)
 
         void PurchaseButton_Click(object sender, EventArgs e)
         {
-            Process.Start("http://particular.net/licensing");
+            Process.Start("https://particular.net/licensing");
         }
 
         void getTrialLicenseButton_Click(object sender, EventArgs e)
         {
-            if (CurrentLicense != null && CurrentLicense.IsTrialLicense)
+            var parameterCollection = new NameValueCollection
```

Comment:
I disagree on the usage of Dictionary instead of NameValueCollection. I don't believe there's a huge readability hit in using one over the other. However, I am going to make it a simple string builder formatted code based on @SimonCropp's suggestion.